docs/vue-integration.asciidoc

```diff
@@ -102,8 +102,31 @@ export default {
 </script>
 ----
 
+or via the recommended `<script setup>` syntax
```

Comment:
curious: Is this the recommended way in Vue?

Reply:
Hi @vigneshshanmugam. It is the [recommened](https://vuejs.org/api/sfc-script-setup.html) way for those who are using the SFC and Composition API. I can amend it if you want.

According the CLA: I followed the [link](https://www.elastic.co/de/contributor-agreement) and completed the instructions. Did I miss something?

Reply:
@Lightwight It would be nice if you could amend it. The way I see it, it will add more context in the documentation 👍 

An according to CLA, you should be sure that the email used in the CLA it's the same one you used to do the commit. Perhaps this thread helps you:  https://discuss.elastic.co/t/signed-cla-yet-pr-shows-as-not-signed/221715/13

Btw, thanks a lot for your contribution to Elastic

Reply:
@devcorpio Thank you for your response and sorry for the expense.
I commited the updated documentation and resigned the CLA with the correct E-Mail and company information. 
But the CLA check wont update.

Reply:
@Lightwight have you amended the commits to make sure that their email associated and the email used in the CL are the same?

Reply:
It's working now, I run the CLA checker manually

Reply:
@devcorpio Thank you very much. I appreciate it 👍